app/models/manageiq/providers/vmware/infra_manager/refresher.rb

```diff
@@ -10,9 +10,6 @@ class InfraManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       # Development helper method for setting up the selector specs for VC
       def self.init_console(use_vim_broker = false)
```

Comment:
Is the parameter to this method even needed anymore?  The whole point of the parameter was to allow an override, but you are now doing that with a setting.